docs/device_api.md

```diff
@@ -314,30 +328,27 @@ The message payload contains the plain attribute value ('5.4').
 
 In the case of multiple measurements, the MQTT message will contain a JSON Object with multiple attributes, each one
 indicating the value of a single measurement, as illustrated in the following example:
+
 ```
 $ mosquitto_pub -h $HOST_IOTAGENT_MQTT -u theUser -P thePassword -t /<myapikey>/<mydevicemqtt>/attrs -m '{ "t": 5.4, "o": 4.3, "n": 3.2, "c": 2.1 }'
 ```
 
 As it can be noticed in this example, the MQTT broker uses a set of credentials to authenticate users. Please, if you
 don't know your credentials, please ask the support team to provide you with a new set.
 
-The topic parts are the same as in the case of a single measure, excluding the measurement name. In this case, four
+The topic parts are the same as in the case of a single measure, excluding the measurement name. In this case, four 
 measurements will be updated in the target entity.
 
 # Act upon devices 
 
-## Transport protocol
-In order to send commands to devices, you just need to know which attributes correspond to commands and update them.
-You can declare the command related attributes at the registry process (as shown in the previous "Register your IoT
-device" section).
+## Send commands
 
-If you take a look to the previous device example, you can find that a "ping" command was defined.
-Any update on this attribute “Ping” at the NGSI entity in the ContextBroker will send a command to your device.
-For instance, to send the "Ping" command with value "Ping request" you could use the following operation in the
-ContextBroker API:
+In order to send commands to devices, you need to know which attributes correspond to commands and update them. You can declare the command related attributes at the registry process (as shown in the previous "Register your IoT device" section). Also, you can declare the protocol you want the commnads to be sent (HTTP/MQTT) with the "transport" parameter at the registry process.
+
+If you take a look to the previous device example, you can find that a "ping" command was defined. Any update on this attribute “ping” at the NGSI entity in the ContextBroker will send a command to your device. For instance, to send the "ping" command with value "Ping request" you could use the following operation in the ContextBroker API:
 
 ```
-PUT /v2/entities/[ENTITY_ID]/attrs/ping
+PUT /v2/entities/<entity_id>/attrs/ping?type=<entity_type>
```

Comment:
Added entity type to the request. This way it works for sure

Reply:
check with Fermin if this is mandatory or a CB glitch. Meanwhile state that type is mandatory.

Reply:
(create an issue for this)

Reply:
This is exepected behaviour. From https://fiware-orion.readthedocs.io/en/master/user/v1_v2_coexistence/index.html#ngsiv2-query-update-forwarding-to-context-providers:

> On forwarding, any type of entity in the NGSIv2 update/query matches registrations without entity type. However, the opposite doesn't work, so if you have registrations with types, then you must use `?type` in NGSIv2 update/query in order to obtain a match.